tests/cache/test_region.py

```diff
@@ -534,6 +536,16 @@ def test_should_delete_multiple_values(self):
         eq_(NO_VALUE, reg.get("key2"))
         eq_(values["key3"], reg.get("key3"))
 
+    def test_get_age(self):
+        reg = self._region()
+        reg.set("some key", "some value")
+        time.sleep(1)
```

Comment:
so I know the test suite has a lot of time.sleep() in it but technically we should not be using it for most tests, instead the time should be set directly.  this both prevents slowing down the suite and additionally eliminates guesswork like "approx".   for example see https://github.com/sqlalchemy/dogpile.cache/blob/efa73d3288e18cc80d8fd484a7ee27c894b7d760/tests/test_lock.py#L299



here you can change this test like this:

```py

with mock.patch(time.time, 100):
    reg.set("some key", "some value")
with mock.patch(time.time, 105):
   eq_(reg.get_age("some key"), 5)

```

note our "eq_" generally places the "thing to be tested" as the first argument.

Reply:
I totally agree (I was a bit surprised by existing tests), thanks for the mock example.